lib/godmin/engine_wrapper.rb

```diff
@@ -17,7 +17,7 @@ def namespaced?
     def namespaced_path
       @namespaced_path ||= begin
         if namespaced?
-          namespace.name.classify.split("::").map(&:underscore)
+          namespace.name.split("::").map(&:underscore)
```

Comment:
Was there a reason for running `classify` here? The namespace should always be a class, right? So it should be "classified" already. Or is it possible for namespace to be something else?